mobile/src/modules/price-api/store/reducer.ts

```diff
@@ -52,6 +63,19 @@ export interface HistoricalPriceInfo {
   HasWarning?: boolean;
 }
 
+export enum PriceType {
+  BURST,
+  BTC,
+  USD
+}
+export type PriceTypeStrings = keyof typeof PriceType;
+
+export enum HistoricalPriceTypes {
+  BTC,
+  USD
+}
```

Comment:
A bit confusing here, but basically the user has 3 options in the app, but only two of those use historical data (as 1 BURST = 1 BURST).

Reply:
looks like you should add a // comment